### PR TITLE
Implement 1 to master

### PR DESCRIPTION
**Pull request by Aichnerc**
**Checklist:**

 
 - [x] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [ ] Have you updated or added documentation for the change?
 - [x] Have you tested your changes with successful results?

**Type of Changes:**

 
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [ ] New feature (non-breaking change which adds functionality)
 - [ ] Documentation (non-breaking change which adds documentation)
 - [x] Breaking change (fix or feature that would cause existing functionality to change)

**What is the current behavior? (link to any open issues here)**
- Currently there is no Shopify client Apollo GraphQL Redux.
- Currently there are no Shopify actions, queries and reducer.

**What is the new behavior (if this is a feature change)?**
- The Shopify client Apollo GraphQL Redux setup has been added.
- The Shopify actions, queries and reducer have been implemented. Shopify has been initialized.

**Other information**:
- The version has been set to `v1.0.0`.

### CODE REVIEW

**Review by kleberbaum — APPROVED** [review]
Should be ok to merge.

**Review by sweggerat — APPROVED** [review]
looks good to meh

File: src/index.js [hunk 1]
```diff
@@ -48,6 +52,24 @@ import fbInit from "./config/fbInit";
 //#endregion
 
 //#region > Config
+//> Shopify API
+// API Link
+const httpLinkShopify = createHttpLink({
+  uri: `https://${process.env.REACT_APP_SHOP}.myshopify.com//api/graphql`,
+});
+// Storefront access token
```
Comment (Theralun) [inline src/index.js:61]: new line before.

**Review by Theralun — CHANGES_REQUESTED** [review]
Minor discomfort.

**Review by Theralun — COMMENTED** [review]
Will look ok soon.

**Review by Theralun — APPROVED** [review]
Looks ok.

**Pull request merged**